english/watagatapistsusberry/watagatapitsusberry_Fruit.rtf

```diff
@@ -0,0 +1 @@
+Berry with cranberry, you're feeling so hairy, maybe you're spaghetti? Those are the side effects of eating the watagatapitsusberry fruit! 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.